.github/workflows/ci_cpu.yml

```diff
@@ -25,22 +25,27 @@ jobs:
         with:
           python-version: 3.8
           architecture: x64
-      - name: Setup Java 11
+      - name: Setup Java 17
         uses: actions/setup-java@v2
         with:
           distribution: 'zulu'
-          java-version: '11'
+          java-version: '17'
       - name: Checkout TorchServe
         uses: actions/checkout@v2
       - name: Install dependencies
         run: |
           python ts_scripts/install_dependencies.py --environment=dev
       - name: Torchserve Sanity
-        run: | 
+        run: |
           python torchserve_sanity.py
       - name: mvn install
-        run: cd serving-sdk/ && mvn clean install -q && cd ../
-        
+        run: |
+            if [[ ${{ matrix.os }} = 'ubuntu-latest' ]]; then
```

Comment:
Could you please add a line comment as to why this `if` condition is needed

Reply:
yes, java is not installed on windows from install_dependecies. I installed them manually and posted the logs.

Reply:
Ran `.\mvnw.cmd clean install -q` on a windows server manually and it works.